service/src/main/java/org/sunbird/dao/otp/impl/OTPDaoImpl.java

```diff
@@ -160,19 +160,11 @@ public Map<String, Object> getOTPDetailsV3(String type, String key, RequestConte
     return otpMapList.get(0);
   }
 
-  /**
-   * Overrides the method to update OTP (One-Time Password) details based on the provided parameters map.
-   * This method delegates the update operation to the Cassandra database operation.
-   *
-   * @param parametersMap A map containing parameters for updating OTP details.
-   *                      It may include information such as type, key, and context token.
-   * @param requestContext The request context associated with the update operation,
-   *                       providing contextual information for the update process.
-   */
   @Override
-  public void updateOTPDetailsV3(Map<String, Object> parametersMap, RequestContext requestContext) {
-    // Delegate the update operation to the Cassandra database operation
-    cassandraOperation.upsertRecord(JsonKey.SUNBIRD, TABLE_NAME, parametersMap, requestContext);
+  public void updateOTPDetailsV3(String keyspaceName, String tableName, Map<String, Object> request, Map<String, Object> compositeKey, RequestContext context) {
+    String expirationInSeconds = PropertiesCache.getInstance().getProperty(JsonKey.OTP_EXPIRATION_TIME_TOKEN);
+    int ttl = Integer.valueOf(expirationInSeconds)/1000;
```

Comment:
if the variable is in seconds -- you don't need to divide by 1000 - right ?

this will create confusion... just keep the value as seconds only - since other variables are also in seconds...
in external properties file -> do update the value to seconds if it is not.